### ISSUE TITLE
Implement state data for lwAFTR

### ISSUE THREAD

**Issue by wingo** [Igalia/snabbswitch#561]
In the `yang` branch we have switched the lwaftr configuration over to use the new YANG code; sweet!  We have ongoing work there to be able to reconfigure a running app, but things are going well.

One important next piece to implement is the ability to respond to queries in general, and specifically to be able to read counters and other state data.  So @xray7224 if you are willing, let's do this :)  Here are some possible steps to victory:

- [ ] Add a state data tree to the snabb-softwire-v1 schema.  Include all counters.

- [ ] Implement a function that takes an instance name and a yang schema name and builds up a Lua object which is a valid instance of the state part of the snabb-softwire-v1 schema.  To do this, you'll be grovelling the filesystem for counters.

- [ ] Add "snabb config get" program with the external interface described in https://github.com/snabbco/snabb/issues/987.  It has three parameters: the YANG schema for your query, the name of the snabb instance, and a path to the data.  Call the function you implemented above to get the data, then assert that the path is `/`, then print the data via `yang.print_data_for_schema_name`.

- [ ] Add the ability to select sub-paths via #549.  In some more perfect world, selecting a path would allow you to avoid generating the whole state data, but as a first step this is probably OK.

**Comment by wingo** [comment]
The second step is the only lwaftr-specific one really.  So that we can keep this in `snabb config` and be somewhat generic, let's resolve a YANG support module via

``` lua
function support_module_for_schema_name(schema_name)
   -- FIXME: @ is not valid in a Lua module name.
   -- if revision then name = name .. '@' .. revision end
   name = name:gsub('[^%w_]', '_')
   return require('lib.yang.support.'..name)
end
```

The `build_state_data` function or something would then be looked up in that module.

**Comment by tsyesika** [comment]
This can be closed, it was implemented as of PR #593
